js/report.js

```diff
@@ -141,6 +185,106 @@ const doc_keyUp = e => {
             break;
     }
 }
-
 // register the handler
 document.addEventListener('keyup', doc_keyUp, false);
+
+
+/**
+ * Remove block overlay
+ */
+const remove_block_overlay = () => {
+    let map = document.getElementById('block-overlay-map');
+    map.innerHTML = "";
+    // remove loading block data
+    let children = document.getElementsByTagName("head")[0].children;
+    for (let i = 0; i < children.length; i++) {
+        if (children[i].src && children[i].src.includes('/report/files/blocks')) {
```

Comment:
Its better to search for elements by tagname.

Can you fix this in a follow up.